src/content/docs/en/reference/modules/astro-content.mdx

```diff
@@ -149,7 +210,35 @@ const draftBlogPosts = await getCollection('blog', ({ data }) => {
 ---
 ```
 
-[See the `Content Collection` guide](/en/guides/content-collections/#querying-collections) for example usage.
+[See the `Content Collection` guide](/en/guides/content-collections/#querying-build-time-collections) for example usage.
+
+### `getLiveCollection()`
+
+<p>
+
+**Type:** `(collection: string, filter?: (entry: CollectionEntry<collection>) => boolean) => CollectionEntry<collection>[]`
+<Since v="5.0.0" />
+</p>
+
+`getLiveCollection()` is a function that retrieves a list of live content collection entries by collection name.
+
+It returns all items in the collection by default, and accepts an optional `filter` function to narrow by entry properties. This allows you to query for only some items in a collection based on `id` or frontmatter values via the `data` object.
```

Comment:
This isn't how live filters work. They're objects that are passed to the loader, which can use them as they want. See [the experimental flag docs](https://docs.astro.build/en/reference/experimental-flags/live-content-collections/#type-safe-filters) for some info. lmk if that's unclear